conformance/reports/v1.0.0/kong-kubernetes-ingress-controller.yaml

```diff
@@ -1,41 +1,41 @@
 apiVersion: gateway.networking.k8s.io/v1alpha1
-date: "2023-11-23T09:42:40Z"
+date: "2024-01-11T10:57:53Z"
 gatewayAPIVersion: v1.0.0
 implementation:
   contact:
-    - github.com/kong/kubernetes-ingress-controller/issues/new/choose
+  - github.com/kong/kubernetes-ingress-controller/issues/new/choose
   organization: Kong
   project: kubernetes-ingress-controller
   url: github.com/kong/kubernetes-ingress-controller
-  version: v3.0.1
+  version: v3.0.2
```

Comment:
@mlavacca @shaneutt Any ideas for how we can show that multiple versions of a product are conformant with a Gateway API release?

Reply:
I have a GEP update PR opened for a while that is actually addressing this exact point. It has already been approved by @shane and is waiting for lgtm. here is the PR: https://github.com/kubernetes-sigs/gateway-api/pull/2391
@robscott could you please take a look at it?